netkan/netkan/metadata.py

```diff
@@ -340,14 +342,8 @@ def source_download(self, branch: str = 'master') -> Optional[str]:
 
     def authors(self) -> List[str]:
         auth = self.author
-        if isinstance(auth, list):
-            return auth
-        else:
-            return [auth]
+        return auth if isinstance(auth, list) else [auth]
 
     def licenses(self) -> List[str]:
         lic = self.license
-        if isinstance(lic, list):
-            return lic
-        else:
-            return [lic]
+        return lic if isinstance(lic, list) else [lic]
```

Comment:
Assume we have code coverage for this file?

Reply:
If you mean tests, yes there are a ton in `netkan/tests/metadata.py`. These are the assertions for both use cases of this property:

https://github.com/KSP-CKAN/NetKAN-Infra/blob/1387cc651a7582e50aa324b963743bc7e52f2e91/netkan/tests/metadata.py#L160-L161

https://github.com/KSP-CKAN/NetKAN-Infra/blob/1387cc651a7582e50aa324b963743bc7e52f2e91/netkan/tests/metadata.py#L206-L207